docker_templates/templates/docker_images/create_drcsim_image.Dockerfile.em

```diff
@@ -18,24 +18,46 @@
     os_name=os_name,
     os_code_name=os_code_name,
 ))@
+@{
+packages = [
+    'dirmngr',
+    'gnupg2',
+    'lsb-release'
+]
+if 'upstream_packages' in locals():
+    if isinstance(upstream_packages, list):
+        for pkg in upstream_packages:
+            if pkg not in packages:
+                packages.append(pkg)
+}@
```

Comment:
Perhaps the packages could also be alpha sorted after all are appended.

Reply:
I considered it and like the fact that this allowed to see the difference between user provided packages and the ones hard coded in the templates so I didn't do it.
I dont feel very strongly either way

Reply:
I still think sorting them would help readability of the dockerfiles, keeping them consistently formatted. And if the delta added by the template wasn't immediately identifiable, one could always check the template.